src/Uno.UWP/Storage/ApplicationData.reference.cs

```diff
@@ -1,23 +1,22 @@
 using System;
 using System.IO;
 
-namespace Windows.Storage
+namespace Windows.Storage;
+
+partial class ApplicationData
 {
-	partial class ApplicationData
-	{
-		private static string GetLocalCacheFolder()
-			=> Path.GetTempPath();
+	private static string GetLocalCacheFolder()
+		=> Path.GetTempPath();
```

Comment:
The standard on *nix is to follow FreeDesktop.org's XDG base directory spec. The spec defines environment variables for cache folders, local folders, etc. Usually, apps follow the environment variable if it exists and fall back to very common values if not (.e.g ~/.cache for cache and ~/.local/share for local, etc.). Arch wiki provides a good summary here: https://wiki.archlinux.org/title/XDG_Base_Directory.